CONTRIBUTING.md

```diff
@@ -68,6 +68,14 @@ You'll need the following:
 * [Docker Compose](https://docs.docker.com/compose/install/)
 * [Foundry](https://getfoundry.sh)
 
+### Install the Correct Version of NodeJS
```

Comment:
Can you move this below the clone instructions?   They won't be able to run `nvm use` until after they have already cloned it